.github/workflows/notify.yml

```diff
@@ -0,0 +1,22 @@
+# When a new release is created on GitHub, this workflow sends an event
+# to cross-platform SDK repos that depend on native iOS SDK
+name: Notify
```

Comment:
Maybe we `name` this `Downstream version bump`?  And name the file `downstream-version-bump.yml`?